src/common/utils/logo.ts

```diff
@@ -0,0 +1,17 @@
+// https://stackoverflow.com/questions/3971841/how-to-resize-images-proportionally-keeping-the-aspect-ratio
+interface GetDimensionsI {
+  width: number;
+  height: number;
```

Comment:
want to copy the unit test over too? https://github.com/TradeTrust/generic-templates/blob/7ad44c3980164716f213e583ae066df9bb469e22/src/common/utils/index.test.ts

Reply:
Sure, thanks Boon!